vars-qa.yml

```diff
@@ -9,6 +9,7 @@ google-analytics-measurement-id:
 dfe-analytics-dataset: fatp_events_qa
 govuk-notify-enquiry-submitted-confirmation-to-enquirer-template-id: b1e29214-dfc0-4225-b65a-df6a20c4c297
 govuk-notify-enquiry-submitted-to-tp-template-id: b24ffca7-6119-4dbf-8ae2-dceb2c300ca4
+govuk-notify-enquiry-response-received-confirmation-to-enquirer-template-id: f64db52d-08ce-4f2a-9b37-a374db1ad77a
```

Comment:
When adding in the other templates IDs I've also added them to the other var yml files (staging, research, production etc).  Is it worth adding this one in to all vars too?  If not then you'll just need to remember to do it at some point in the future.

Reply:
Cool - I have added that for the rest of the `yml` files (staging, research, production).